Update audio.md

nmaruy27 opened the pull request:
Since createNewLoadedSound has been deprecated, it has been modified to use the alternative createNewLoadedSoundAsync.

nmaruy27 committed:
Update audio.md

Since createNewLoadedSound has been deprecated, it has been modified to use the alternative createNewLoadedSoundAsync.

cruzach commented:
Great! Thanks for fixing this 😄 

Could you make the same change to the `unversioned` docs, as well? 
https://github.com/expo/expo/blob/master/docs/pages/versions/unversioned/sdk/audio.md

nmaruy27 commented:
Thank you for reply.

What I fixed is unversioned.
Are you talking about other versions? For example, v34.0.0, v33.0.0, etc.

cruzach commented:
Yeah sorry for the confusion, I meant v33 and v34

nmaruy27 committed:
Update audio.md

Since createNewLoadedSound has been deprecated, it has been modified to use the alternative createNewLoadedSoundAsync.

nmaruy27 committed:
Update audio.md

Since createNewLoadedSound has been deprecated, it has been modified to use the alternative createNewLoadedSoundAsync.

nmaruy27 commented:
@cruzach 
OK. I made the same change to v33 and v34.
